tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -257,10 +257,10 @@ def add_rancher_images_to_private_registry(bastion_node, push_images=True):
             RANCHER_SERVER_VERSION)
     bastion_node.execute_command(get_images_command)
 
-    # Remove the "docker save" and "docker load" lines to save time
+    # comment out the "docker save" and "docker load" lines to save time
     edit_save_and_load_command = \
-        "sudo sed -i '58d' rancher-save-images.sh && " \
-        "sudo sed -i '76d' rancher-load-images.sh && " \
+        "sudo sed -i -e 's/docker save /# docker/g' rancher-save-images.sh && " \
+        "sudo sed -i -e 's/docker load /# docker/g' rancher-load-images.sh && " \
```

Comment:
NIT: To truly comment out, we should probably make these:
```
        "sudo sed -i -e 's/docker save /# docker save /g' rancher-save-images.sh && " \
        "sudo sed -i -e 's/docker load /# docker load /g' rancher-load-images.sh && " \
```
otherwise, in the `rancher-save-images.sh` for example, it will be like:
```
# docker$(echo ${pulled}) | gzip --stdout > ${images}
```